Contribute/get-started-setup-local.md

```diff
@@ -1,150 +1,217 @@
+   If your current working directory is not the cloned repository, change directory (`cd`) to it. You can also test if you are in a repository by running `git status`.
 
-## Configure remote upstream
-After cloning the repository, set up a read-only remote connection to the main repository named **upstream**. You use the upstream URL to keep your local repository in sync with the latest changes made by others. The **git remote** command is used to set the configuration value. You use the **fetch** command to refresh the branch info from the upstream repository.
+2. Add the Microsoft repository as an upstream repository.
 
-1. If you're using **Git Credential Manager**, use the following commands. Replace the \<repo\> and \<organization\> placeholders.
    ```bash
-   cd <repo>
-   git remote add upstream https://github.com/<organization>/<repo>.git
-   git fetch upstream
+   git remote add upstream https://github.com/MicrosoftDocs/<Repo-Name>.git
    ```
 
-2. View the configured values and confirm the URLs are correct. Ensure the **origin** URLs point to your personal fork. Ensure the **upstream** URLs point to the main repository, such as MicrosoftDocs or Azure. 
+3. Validate that the remote URLs are correct.
+
    ```bash
-   git remote -v 
+   git remote -v
    ```
 
-   Example remote output is shown. A fictitious git account named MyGitAccount is configured with a personal access token to access the repo azure-docs:
+   The **origin** URL entires should be your fork URL, while the **upstream** URL entires should be the Microsoft repository URL.
+
    ```output
-   origin  https://github.com/MyGitAccount/azure-docs.git (fetch)
-   origin  https://github.com/MyGitAccount/azure-docs.git(push)
-   upstream        https://github.com/MicrosoftDocs/azure-docs.git (fetch)
-   upstream        https://github.com/MicrosoftDocs/azure-docs.git (push)
+   origin  https://github.com/<Username>/<Repo-Name>.git (fetch)
+   origin  https://github.com/<Username>/<Repo-Name>.git (push)
+   upstream        https://github.com/MicrosoftDocs/<Repo-Name>.git (fetch)
+   upstream        https://github.com/MicrosoftDocs/<Repo-Name>.git (push)
    ```
 
-3. If you made a mistake, you can remove the remote value. To remove the upstream value, run the command `git remote remove upstream`.
+   `<Username>` will be your GitHub username, or the GitHub account/organization you chose when forking the Microsoft repository. `<Repo-Name>` will be the respective fork and Microsoft repository name. The Microsoft repository name and your fork repository name may be different depending on your choices when forking.
+
+4. If you made a mistake, you can remove the remote upstream value.
+
+   ```bash
+   git remote remove upstream
+   ```
+
+5. Once your upstream repository is correctly configured you can fetch changes from the upstream repository at any time.
```

Comment:
Fetch will get a list of branches updated. In the SVG graphic, we usually tell people to pull from upstream, and not pull from their out of date forks.